### PR TITLE
add new recipes for spectrum

### PR DESCRIPTION
**Pull request by cokelaer**

### CODE REVIEW

File: recipes/spectrum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+{% set name = "spectrum" %}
+{% set sha256 = "7d8e3fe074b2026f61b6539c4f847d0cb1dbed41d5c6788f6100bb5884aa65ec" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/a4/c85957430db53744fd70ce726d8cd0000ee33e18da39a8132ca4e5cdb2f5/{{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/spectrum/meta.yaml]: Can you use the following canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

File: recipes/spectrum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0 
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/spectrum/meta.yaml]: It looks like only `python`, `setuptools` and `toolchain` are required for the `build` due to using `--single-version-externally-managed`

File: recipes/spectrum/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - scipy
+    - easydev
+
+test:
+  imports:
+    - spectrum
+
+about:
+  home: http://github.com/cokelaer/spectrum
+  license: LGPL v3+
```
Comment (synapticarbors) [inline recipes/spectrum/meta.yaml]: The license file in the github repo appears to be `BSD-3-clause`, although the meta-data in the `setup.py` says LGPL.

File: recipes/spectrum/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  run:
+    - python
+    - matplotlib
+    - numpy
+    - scipy
+    - easydev
+
+test:
+  imports:
+    - spectrum
```
Comment (synapticarbors) [inline recipes/spectrum/meta.yaml]: Not sure if this checks the presence of the C-extension. If not, maybe add something to ensure that it is present. I'm not familiar with the library, but maybe there is a simple import test.

File: recipes/spectrum/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+about:
+  home: http://github.com/cokelaer/spectrum
+  license: LGPL v3+
+  license_family: LGPL
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/spectrum/meta.yaml]: It doesn't look like the license file is packaged with the source distribution. Please include it manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and put in either an issue or a pull request upstream to include it via the `MANIFEST.in`. Please add the url for the issue as a comment in the recipe and xref the link here for future reference when upstream gets updated.

File: recipes/spectrum/meta.yaml (hunk 1 above)
Comment (cokelaer) [inline recipes/spectrum/meta.yaml]: fixed

File: recipes/spectrum/meta.yaml (hunk 2 above)
Comment (cokelaer) [inline recipes/spectrum/meta.yaml]: fixed

File: recipes/spectrum/meta.yaml (hunk 4 above)
Comment (cokelaer) [inline recipes/spectrum/meta.yaml]: added a import spectrum.mtm that imports the C extension

File: recipes/spectrum/meta.yaml (hunk 3 above)
Comment (cokelaer) [inline recipes/spectrum/meta.yaml]: fixed in the recipes

File: recipes/spectrum/meta.yaml (hunk 5 above)
Comment (cokelaer) [inline recipes/spectrum/meta.yaml]: fixed in the new version 0.6.5 updated in the recipes

**Pull request merged**